week2/homework/src/index.js

```diff
@@ -1,3 +1,128 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+
+const command = process.argv[2];
+const command2 = process.argv[3];
+const command3 = process.argv[4];
+
+fs.readFile('./to-do.json', 'utf8', (err, data) => {
```

Comment:
I changed my code. Now if there is no to-do.json file, the application will create a to-do.json file. The code about it is inside `fs.readFile('./to-do.json', ...`
It may be a little complicated but it works very well. 
Thank you @J05J4